iceoryx_posh/include/iceoryx_posh/internal/roudi/service_registry.hpp

```diff
@@ -80,6 +81,20 @@ class ServiceRegistry
               const cxx::optional<capro::IdString_t>& instance,
               const cxx::optional<capro::IdString_t>& event) const noexcept;
 
+    /// @brief Searches for given service description in registry and applies a function to each
+    ///        matching entry, the function may have side effects but cannot change the entry
+    /// @param[in] searchResult, reference to the vector which will be filled with the results
+    /// @param[in] service, string or wildcard (= iox::cxx::nullopt) to search for
+    /// @param[in] instance, string or wildcard (= iox::cxx::nullopt) to search for
+    /// @param[in] event, string or wildcard (= iox::cxx::nullopt) to search for
+    /// @param[in] function, the function to apply to the matching entries
```

Comment:
Since the doxygen docu is identical to the `service_discovery.hpp` why not use `\\\ @copydoc ServiceDiscovery::findService`?